craft_parts/plugins/validator.py

```diff
@@ -33,8 +33,14 @@ class PluginEnvironmentValidator:
     deb package, a snap, built from sources or even built by a different
     part. Plugin environment validators allow a plugin to ensure its
     execution environment is correct before building a part.
+
+    :param part_name: The part whose build environment is being validated.
+    :param env: A string containing the build step environment setup.
     """
 
+    COMMAND_NOT_FOUND = 127
```

Comment:
is this here due to a missing rebase or something?

Reply:
I moved the definition from each validator implementation to the base class, but perhaps it would be better to have it defined in package level.